src/Symfony/Component/Console/Completion/CompletionInput.php

```diff
@@ -179,6 +181,18 @@ public function mustSuggestArgumentValuesFor(string $argumentName): bool
         return self::TYPE_ARGUMENT_VALUE === $this->getCompletionType() && $argumentName === $this->getCompletionName();
     }
 
+    public function setShell(string $shell): string
+    {
+        [$this->shell, $old] = [$shell, $this->shell];
+
+        return $old;
```

Comment:
The "old" value is never used.

```suggestion
    public function setShell(string $shell): void
    {
        $this->shell = $shell;
```